octopusCats/src/main/scala/octopus/cats/package.scala

```diff
@@ -0,0 +1,23 @@
+package octopus
+
+import _root_.cats.data.{Validated, ValidatedNel, NonEmptyList}
+
+package object cats {
+
+  implicit class OctopusCatsOps[T](val vr: ValidationResult[T]) extends AnyVal {
+
+    def toValidatedNel: ValidatedNel[ValidationError, T] = vr.errors match {
+      case Nil =>
+        Validated.Valid(vr.value)
+      case head :: tail =>
+        Validated.Invalid(NonEmptyList.of(head, tail : _*))
+    }
+
+    def toValidated: Validated[List[ValidationError], T] = vr.errors match {
```

Comment:
Tried that, could work as well, but with pattern match I don't have to transform errors in any way and can construct NEL instance directly from extracted `head` & `tail`.